recipes/deepxde/meta.yaml

```
@@ -0,0 +1,54 @@
+{% set name = "DeepXDE" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 8326390e658d93f2412e3173f3834996502d61abbac64be982e7b5b455bfdc3a
+
+build:
+  number: 0
+  # Skip windows with Pytho 2.7 as TensorFlow does not support it well.
+  skip: True  # [win and py27]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - numpy
+  run:
+    - python
+    - {{ pin_compatible('numpy') }}
+    - matplotlib
+    - salib
+    - scikit-learn
+    - scipy
+    - tensorflow
+
+test:
+  imports:
+    # Skip the import test on Linux as TensorFlow require a more recent
+    # version of GLIBC++ than the VM used to build and test package.
+    - deepxde  # [not linux]
+
+about:
+  home: https://github.com/lululxvi/deepxde
+  license: Apache-2.0
+  license_family: Apache
```

Add the license_file back.  Apache licenses require redistribution of the license

```suggestion
  license_family: Apache
  license_file: LICENSE
```

---

Resolved.

---

@lululxvi You only need the entry in the meta.yaml, as you did... no need to add the license file itself into the PR. It will be picked out of the pypi sdist tarball.

---

That was the reason to put it into the pypi tarball to easily refer to it from the recipe.

---

@dbast Thank you. Resolved.